mitol-django-digital-credentials/testapp/README.md

```diff
@@ -0,0 +1,62 @@
+`mitol-django-digital-credentials` test app
+---
+
+#### Setup
+
+To run this app in local development mode, copy `testapp/settings/example.dev.py` to  `testapp/settings/example.dev.py`. This file has the same defaults as `testapp/settings/test.py`, but it is gitignored so you can safely add secrets to it. `manage.py` and `testapp/wsgi.py` both load `dev.py`.
+
+#### Usage
+
+Ensure the [`sign-and-verify`](https://github.com/digitalcredentials/sign-and-verify) service is running locally.
+
+In the `mitol-django-digital-credentials directory`, you can test this app out by running:
+
+- `poetry run python manage.py migrate` - migrates sqlite db (gitignored)
+- `poetry run python manage.py shell`:
+
+```python
+from datetime import timedelta
+from oauth2_provider.models import AccessToken, get_application_model
+from mitol.common.factories import UserFactory
+from mitol.common.utils import now_in_utc
+from mitol.digitalcredentials.factories import DemoCoursewareDigitalCredentialRequestFactory
```

Comment:
when I tried to run this locally this module didn't have `DemoCoursewareDigitalCredentialRequestFactory` but it does have `DigitalCredentialRequestFactory`